test/sortitionPoolTest.js

```diff
@@ -261,7 +274,7 @@ describe("SortitionPool", () => {
   describe("pool rewards", async () => {
```

Comment:
What if we define `withdrawRewards()` function only in this context? It an auxiliary function to test pool rewards and is not used anywhere else.

```
describe("pool rewards", async () => {
    async function withdrawRewards(
      pool,
      owner,
      operatorAddress,
      beneficiaryAddress,
    ) {
      // (...)
    }
// (...)
```